gobblin-core-base/src/main/java/org/apache/gobblin/instrumented/Instrumented.java

```diff
@@ -255,6 +259,10 @@ public static void markMeter(Optional<Meter> meter) {
     markMeter(meter, 1);
   }
 
+  public static void markMeter(Meter meter) {
```

Comment:
where is this being used?

Reply:
my related Q is whether we still require the `Optional` version or could *replace* them w/ this one

Reply:
There are many non-gaas callers to this method with Optional parameter, which may be providing parameter that is truly optional.